### ISSUE TITLE
Question: Wave2Letter+ Configuration for different number of GPUs

### ISSUE THREAD

**Issue by the01** [NVIDIA/OpenSeq2Seq#322]
I was attempting to reproduce the results of the Wave2Letter+ model from [the docu](https://nvidia.github.io/OpenSeq2Seq/html/speech-recognition/wave2letter.html) using [this config](https://github.com/NVIDIA/OpenSeq2Seq/blob/18.09/example_configs/speech2text/w2lplus_large_8gpus.py) as basis. I only have 2 Titan X available, so I couldn't use mixed-precision and changed the number of GPUs to 2.
I was only able to achieve a WER of around 16%, far from the reported 6.67%.

I assume I need a different learning rate/.. for training with 1, 2, 4, .. GPUs? What configuration should I use to get similar result?

**Comment by borisgin** [comment]
Please reduce LR to 0.01 and turn off LARC. 
What was your training ""num_epochs"?

**Comment by the01** [comment]
I assume removing "larc_params" from base_params turns it off?
I believe it was 200 or 250. Do I need a higher value?
For how many GPUs is this setting intended?

**Comment by borisgin** [comment]
This learning rate is for  2 GPUs. Yes, you can remove larc parameters.

**Comment by the01** [comment]
Ok, I changed LR, removed larc_params and turned horovod off, but it now throws ```W tensorflow/core/util/ctc/ctc_loss_calculator.cc:144] No valid path found.``` errors. Do I need to compile tensorflow myself and use the ctc with lm?
I noticed that the ```Wave2LetterEncoder``` was changed to ```TDNNEncoder```.

**Comment by vaibhav0195** [comment]
hi i am also getting this warning 
**tensorflow/core/util/ctc/ctc_loss_calculator.cc:144] No valid path found**
but i am positive that i compiled the tensorflow with the kenlm configuration as specified in the docs.

does this affect training or this is the warning just during when the output is decoded for the transcription ?

**Comment by borisgin** [comment]
Tensorflow is ok, this means that there training diverged, or that maybe audio sequence is too short

**Comment by vaibhav0195** [comment]
oh so that means my model is not being trained ?
